Makefile

```diff
@@ -105,18 +105,20 @@ build-proto-image:
 
 .PHONY: release
 release:
-	$(info Do you want to continue?)
-	$(info Version will be: $(version))
+	$(info Do you want to continue? Version will be: $(version))
 	read
 
 	git checkout develop
 	git pull
 	git checkout -b "release/$(version)"
 
-# ## Create changelog
+## Create changelog
 	git-chglog --next-tag $(version) -o CHANGELOG.md
 	git add CHANGELOG.md
 
+## Change the golang version
+	sed 's/Version *=.*/Version        = "$(version)"/' golang/internal/version/version.go > temp_file && mv temp_file golang/internal/version/version.go
```

Comment:
Need whitespace because of the const format in Golang file.

Reply:
I'd make changes to the version.go file, so if you add/remove a variable  from that, it won't shift around because of the automatic formatting.